worlds/alttp/Client.py

```diff
@@ -681,14 +679,13 @@ def onButtonClick(answer: str = 'no'):
             choice = 'yes'
 
         if 'yes' in choice:
-            import LttPAdjuster
             from .Rom import get_base_rom_path
             last_settings.rom = romfile
             last_settings.baserom = get_base_rom_path()
             last_settings.world = None
 
             if last_settings.sprite_pool:
-                from LttPAdjuster import AdjusterWorld
+                from worlds.alttp.Adjuster import AdjusterWorld
```

Comment:
files in an apworld should import from themselves using relative imports
```suggestion
                from .Adjuster import AdjusterWorld
```